src/tools/cargotest/main.rs

```diff
@@ -120,12 +147,31 @@ fn clone_repo(test: &Test, out_dir: &Path) -> PathBuf {
     out_dir
 }
 
-fn run_cargo_test(cargo_path: &Path, crate_path: &Path, packages: &[&str]) -> bool {
+fn run_cargo_test(
+    cargo_path: &Path,
+    crate_path: &Path,
+    packages: &[&str],
+    features: Option<&[&str]>,
+    manifest_path: Option<&str>,
+) -> bool {
     let mut command = Command::new(cargo_path);
     command.arg("test");
+
+    if let Some(path) = manifest_path {
+        command.arg(format!("--manifest-path={}", path));
```

Comment:
Why not split it into 2 args so we don't need to use `format!`?

Reply:
I think its fine the way it is, or at least, I wouldn't change it based on trying to avoid calls to `format!` in this code.

(On the other hand, it is possible we have a policy somewhere about whether we try to bias when choosing between `--flag value` or `--flag=value` in the command lines we generate. If such a policy exists, then that should be what dictates the choice here...)